### DISCUSSION TITLE
IntelliJ IDEA issue while loading coil project in Android Studio

### DISCUSSION THREAD

**Discussion by halizton** [coil-kt/coil#1310] [Q&A] [answered]
I am having the following issue while loading coil project in AS (version attached)

> This version of the Android Support plugin for IntelliJ IDEA (or Android Studio) cannot open this project, please retry with version 2021.2.1 or newer.

![image](https://user-images.githubusercontent.com/12901839/171909149-faa9ff99-4846-4db9-baf2-d8489fd12927.png)

**Comment by connyduck** [comment]
So update your Android Studio?

**Reply by halizton** [reply]
> So update your Android Studio?

Mind sharing which version are you using?

**Comment by connyduck** [comment, marked as answer]
Android Studio Chipmunk | 2021.2.1
